app/views/main_window.py

```diff
@@ -1,7 +1,6 @@
+import os
```

Comment:
Same as above. Import groups should be sorted alphabetically.

Reply:
I just Used Optimize Imports feature in PyCharm and it arranged it

Reply:
`import os` should be moved back to where it was.

Reply:
same as above